ext/net/ops.rs

```diff
@@ -184,6 +206,152 @@ where
   Ok(nwritten)
 }
 
+#[op]
+async fn op_net_join_multi_v4_udp<NP>(
+  state: Rc<RefCell<OpState>>,
+  rid: ResourceId,
+  membership: MulticastMembershipV4,
```

Comment:
Would probably be a bit more performant to use individual parameters for the address and interface: Iterating through objects is a bit of a performance pain.